include/domain_bridge/topic_bridge_options.hpp

```diff
@@ -107,6 +109,34 @@ class TopicBridgeOptions
   TopicBridgeOptions &
   delay(const std::chrono::milliseconds & delay);
 
+  /// Set wait_for_subscription to `value`.
+  /**
+   * When set to true, the domain bridge will wait for a subscription to be available in the
+   * "to domain" before creating the bridge.
```

Comment:
I think we should document the interplay between `wait_for_subscription` and `wait_for_publisher` here. E.g. If both are `true`, then the bridge first waits for a subscription and then for a publisher.

We should also describe QoS matching, e.g. if we're waiting on both a subscription and a publisher, then waht QoS will be used for the bridge?

Reply:
Done in https://github.com/ros2/domain_bridge/pull/52/commits/19835d8811fe87d04d6e5d469b9d8e8ba6900519